public/app/fn-app/create-mfe.ts

```diff
@@ -250,45 +245,17 @@ class createMfe {
       createMfe.logger('Trying to update grafana with theme:', props.mode);
 
       if (props.mode) {
-        dispatch(
```

Comment:
Why?

Reply:
Told you so many times @ziomecka it is not updating anything. When you try to update global state here it looks like state is being updated but actually true resolved promise update the `mfe` automatically. When you use global dispatch here it updates state but our app is using different instance of global store because this class doesn't come under redux provider we are using.